sequencing/variant_annotator/Readme.md

```diff
@@ -1,21 +1,28 @@
-<!-- dx-header -->
-# VCF QC (DNAnexus Platform App)
+# GATK3's VariantAnnotator 
 
-Calls [GATK VariantAnnotator](https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_annotator_VariantAnnotator.php)
 
-This is the source code for an app that runs on the DNAnexus Platform.
-For more information about how to run or modify it, see
-https://wiki.dnanexus.com/.
-<!-- /dx-header -->
+## What does this app do?
 
+This app run's the [GATK VariantAnnotator](https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_annotator_VariantAnnotator.php).
+This tool is meant to annotate the variant calls in a given `VCF` file based on their context.  
+Given an array of `VCF` files, this app will launch a separate sub-job for each `VCF` and run GATK's VariantAnnotator.  Users can request that
```

Comment:
I think the term is "subjob". Also, remove line break

Reply:
Done